sql/item_weapon.sql

```diff
@@ -3975,12 +3975,54 @@ INSERT INTO `item_weapon` VALUES (21056,'tomonari',10,0,0,0,0,2,1,420,118,0);
 INSERT INTO `item_weapon` VALUES (21057,'tomonari_+1',10,0,0,0,0,2,1,407,119,0);
 INSERT INTO `item_weapon` VALUES (21058,'shichishito',10,0,54,54,42,2,1,450,126,0);
 INSERT INTO `item_weapon` VALUES (21059,'ichijinto',10,0,0,0,0,2,1,420,121,0);
+INSERT INTO `item_weapon` VALUES (21060,'mjollnir',11,0,242,242,228,3,1,308,160,0);
 INSERT INTO `item_weapon` VALUES (21061,'mjollnir',11,0,242,242,228,3,1,308,160,0);
-INSERT INTO `item_weapon` VALUES (21062,'yagrush',11,0,242,242,228,3,2,267,133,0);
-INSERT INTO `item_weapon` VALUES (21063,'yagrush',11,0,242,242,228,3,2,267,133,0);
+INSERT INTO `item_weapon` VALUES (21062,'yagrush',11,0,242,242,228,3,3,267,133,0);
```

Comment:
All of them are wrong, both before and after you touched them.

That isn't part of the weapon. Look at the item help-text:

![Yagrush](http://vignette1.wikia.nocookie.net/ffxi/images/2/26/Yagrush99.png/revision/latest?cb=20120713014710)

That is in the aftermath effects, not the items basic stats.